context/app/static/js/components/workspaces/WorkspaceLaunchSuccessToast.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+
+function WorkspaceLaunchSuccessToast(id: number) {
+  return (
+    <Stack spacing={1} sx={{ maxWidth: '22rem' }}>
```

Comment:
`maxWidth` can be provided as a `system` prop: 
```suggestion
    <Stack spacing={1} maxWidth="22rem">
```

- https://mui.com/material-ui/react-stack/#system-props
- https://mui.com/system/properties/